contents/handbook/growth/sales/expansion-and-retention.md

```diff
@@ -127,3 +127,38 @@ We can roll up to half the amount of credit from the original order form to a ne
 These are customers that are happily using PostHog long term, and are neither a churn risk nor likely to have expansion potential. Managing this group is much more automated and taken care of by CSMs, who do things like tracking usage and setting up alerts in Vitally to trigger outreach from us when a customer changes their usage behavior (either up or down). 
 
 An important part of retention here is also to ensure support issues are fixed in a timely manner. We deliberately don't want to invest a huge amount in hands-on customer success here, because that can often paper over cracks in the product experience or quality of our customer support, so staying hands-off here is an intentional strategy. In the future, we will build out this playbook a lot more. 
+
+## Enterprise account management
```

Comment:
It's not really 'Enterprise' per the vibe matrix - while it's a joke that you guys have called yourselves Abnormal AEs, I don't hate it... But also open to other words here! Just not Enterprise.